appveyor.yml

```diff
@@ -1,4 +1,4 @@
-image: Visual Studio 2017
+image: Visual Studio 2019
```

Comment:
Build fails in the old windows, but I haven't such OS to debug, hope someone else can help with that.

https://ci.appveyor.com/project/halfzebra/create-elm-app/builds/38540133/job/7duw5ityiktqj1wf